### PR TITLE
T03 Completed

### PR DESCRIPTION
**Pull request by YoganathanM**

### CODE REVIEW

File: Assignment/Program.cs [hunk 1]
```diff
@@ -1,7 +1,12 @@
...
-      static void Main (string[] args) {
-         Console.WriteLine ("Hello world!");
+﻿for (; ; ) {
+   Console.Write ("Enter input number :");
+   if (ulong.TryParse (Console.ReadLine (), out ulong n)) {
+      ulong firstTerm = 0, secondTerm = 1;
+      for (ulong i = 0; i < n; i++) {
+         Console.WriteLine ($"f{i} : {firstTerm}");
+         ulong thirdTerm = firstTerm + secondTerm;
+         (firstTerm, secondTerm) = (secondTerm, thirdTerm);
```
Comment (varalakshmi-imm) [inline Assignment/Program.cs:8]:
```suggestion        
 (firstTerm, secondTerm) = (secondTerm, firstTerm + secondTerm);
```

File: Assignment/Program.cs [hunk 2]
```diff
@@ -1,7 +1,12 @@
...
       }
-   }
+      break;
```
Comment (varalakshmi-imm) [inline Assignment/Program.cs:10]: Why do you need this break?

File: Assignment/Program.cs [hunk 3]
```diff
@@ -1,7 +1,12 @@
-﻿namespace Assignment {
-   internal class Program {
-      static void Main (string[] args) {
-         Console.WriteLine ("Hello world!");
+﻿for (; ; ) {
+   Console.Write ("Enter input number :");
+   if (ulong.TryParse (Console.ReadLine (), out ulong n)) {
```
Comment (varalakshmi-imm) [inline Assignment/Program.cs:3]: Why ulong?

**Review by varalakshmi-imm — CHANGES_REQUESTED** [review]

File: Assignment/Program.cs (hunk 3 above)
Comment (YoganathanM) [reply Assignment/Program.cs:3]: To know the fibonacci sequence above maximum integer limit

File: Assignment/Program.cs (hunk 2 above)
Comment (YoganathanM) [reply Assignment/Program.cs:10]: Once the Fibonacci series is done for user input the loop should end

File: Assignment/Program.cs (hunk 1 above)
Comment (YoganathanM) [reply Assignment/Program.cs:8]: ok will change

**Review by varalakshmi-imm — APPROVED** [review]
Having the breaks in this way of implementation is little confusing. Try to avoid that for readability and use the condition in the loop and break as we discussed.

**Pull request closed without merging**